### PR TITLE
Loads fonts via the virtual filesystem.

### PR DESCRIPTION
**Pull request by pchote**
Prerequisite to #10703 and oramods.

### CODE REVIEW

File: mods/d2k/mod.yaml [hunk 1]
```diff
@@ -173,7 +173,7 @@ Fonts:
 		Font:./mods/common/FreeSansBold.ttf
 		Size:14
 	Title:
-		Font:d2k:Dune2k.ttf
+		Font:./mods/d2k/Dune2k.ttf
```
Comment (pchote) [inline mods/d2k/mod.yaml]: Note: This temporarily breaks the ability to load d2k from the support directory.  This will be restored by #10703.

**Comment by penev92** [comment]
:+1:

**Comment by obrakmann** [comment]
:+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/2c310c0e5d276ec23b2d8e00212c005fa249f37e)

**Pull request merged**